source/Notes/index.html

```diff
@@ -2,11 +2,13 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
     <title>Markdown Notes App Journal</title>
     <link rel="stylesheet" href="styles.css">
 </head>
 <body>
+    <!-- Add a style attribute to the button -->
+    <button style="float: left;" onclick="window.location.href='../calendar/index.html'">Back to Calendar</button>
```

Comment:
Is it possible to move the style of the button to a CSS file? 

Reason: Separation of Concerns